lms/djangoapps/mobile_api/users/serializers.py

```diff
@@ -34,6 +35,10 @@ def to_representation(self, course_overview):
                 kwargs={'course_id': course_id},
                 request=request
             )
+            discussion_url = reverse(
+                'discussion_course',
+                kwargs={'course_id': course_id},
+                request=request) if course_overview.is_discussion_tab_enabled() else None
```

Comment:
@nasthagiri The code in serializer works perfectly fine. Its the `users/tests.py` that could not access discussion endpoint as these tests are making use of custom `reverse` implementation and tries to load all the endpoints by looking at the `include` in `mobile_api/urls.py` and creates a dictionary against module and api. If we do not add discussion urls in `mobile_api/urls.py` it raises exception that discussion module could not be found. 
The question is why serializer and test are not implemented to make use of same `reverse` method?